README.md

```diff
@@ -1,26 +1,45 @@
 ## Description
-These rules refer to 
+These rules refer to http://developer.android.com/guide/developing/device.html and include many suggestions from the Archlinux and Github Community.
 
-http://developer.android.com/guide/developing/device.html
+If you're developing on Ubuntu Linux, you need to add a udev rules file that contains a USB configuration for each type of device you want to use for development. In the rules file, each device manufacturer is identified by a unique vendor ID, as specified by the ATTR{idVendor} property. For a list of vendor IDs, see USB Vendor IDs, below.
 
-and include many suggestions from the Archlinux and Github Community
+Use this format to add each vendor to the file(example vendor ID is for HTC):  
+`SUBSYSTEM=="usb", ATTR{idVendor}=="0bb4", MODE="0666", GROUP="plugdev"`
+
+The MODE assignment specifies read/write permissions, and GROUP defines which Unix group owns the device node.
+
+Note: The rule syntax may vary slightly depending on your environment. Consult the udev documentation for your system as needed. For an overview of rule syntax, see this guide to writing udev rules. please refer to 51-android.rules in [ubuntu directory](ubuntu)
 
 ## Installation
-If you're developing on Ubuntu Linux, you need to add a udev rules file that contains a USB configuration for each type of device you want to use for development. In the rules file, each device manufacturer is identified by a unique vendor ID, as specified by the ATTR{idVendor} property. For a list of vendor IDs, see USB Vendor IDs, below. To set up device detection on Ubuntu Linux:
-Log in as root and create this file: /etc/udev/rules.d/51-android.rules.
-Use this format to add each vendor to the file:
-SUBSYSTEM=="usb", ATTR{idVendor}=="0bb4", MODE="0666", GROUP="plugdev" 
 
-In this example, the vendor ID is for HTC. 
-The MODE assignment specifies read/write permissions, and GROUP defines which Unix group owns the device node.
-Note: The rule syntax may vary slightly depending on your environment. Consult the udev documentation for your system as needed. For an overview of rule syntax, see this guide to writing udev rules. 
-please refer to 51-android.rules in [ubuntu directory](ubuntu)
-Now execute:
-chmod a+r /etc/udev/rules.d/51-android.rules
+#### For Ubuntu
 
-Try group plugdev rather than plugindev. Then restart udev:
+```sh
+# Clone the repo (replace repo-url with the Github URL)
+git clone <repo-url>
+# Create a sym-link to the rules file
+sudo ln -s ./android-udev-rules/51-android.rules /etc/udev/rules.d/
+# Change file permissions
+sudo chmod a+r /etc/udev/rules.d/51-android.rules
+# Restart UDEV
 sudo udevadm control --reload-rules
```

Comment:
On 14.x + and most other linux rigs these days a simple  unplug  <device> and  re-insert forces the reload of rules and more  new user /dev friendly (imo)